src/styles/antd-overrides/input.scss

```diff
@@ -8,7 +8,6 @@
 .ant-input,
 .ant-input-affix-wrapper,
 .ant-input-group-addon,
-.ant-input-number-input,
```

Comment:
There seemed to be a double focus effect happening. When I had to add a left-side accessory to the number input for the currency, I also needed to add some padding. Because there was a focus on the inner input, it looked whack like this:

**Before:**
<img width="333" alt="Screen Shot 2023-08-01 at 3 38 35 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/6e4d6764-ef58-4975-ae25-f423d1856bd6">

**After:**
<img width="372" alt="Screen Shot 2023-08-01 at 3 38 50 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/6b3d4b33-38d2-4fc4-868b-5c2562ded3ec">

Btw, the normal prefix prop we already have was not ideal because it can be deleted by the user.